presto-main/src/main/java/io/prestosql/sql/planner/PlanOptimizers.java

```diff
@@ -497,6 +527,8 @@ public PlanOptimizers(
                         estimatedExchangesCostCalculator,
                         ImmutableSet.of(new PushPredicateIntoTableScan(metadata, typeAnalyzer))),
                 projectionPushDown,
+                new StatsRecordingPlanOptimizer(optimizerStats, new PredicatePushDown(metadata, typeAnalyzer, true, false)),
```

Comment:
Missing call to RemoveUnsupportedDynamicFilters here is causing [this query](https://github.com/prestosql/presto/pull/2672#issuecomment-605356480) to be 4x slower:

    new RemoveUnsupportedDynamicFilters(metadata),

Reply:
Upon closer examination, performance is the same without the RemoveUnsupportedDynamicFilters. Sorry for the noise.